web/src/engine/predictive-text/worker-thread/src/main/index.ts

```diff
@@ -403,7 +407,7 @@ export default class LMLayerWorker {
    *
    * @param scope A global scope to install upon.
    */
-  static install(scope: DedicatedWorkerGlobalScope): LMLayerWorker {
+  static install(scope: any /*DedicatedWorkerGlobalScope*/): LMLayerWorker {
```

Comment:
What happened to `DedicatedWorkerGlobalScope`?

Reply:
See thread: https://github.com/keymanapp/keyman/pull/14510/files#r2271925303